andela/ah-centauri-frontend
# Chore/166235823 ui update

#### What does this PR do?
- Updates the user interface
#### Description of Tasks to be completed?
- Update the login button
- Update the login modal

#### How should this be manually tested?
- Clone the project: git clone https://github.com/andela/ah-centauri-frontend
- cd into the project folder
- checkout to branch `git checkout chore/166235823-UI-update`
- Install dependencies: npm install
- Launch development server: npm start
- Click on the login button on the header and a modal will appear

#### Any background context you want to provide?

#### What are the relevant Pivotal Tracker Stories?
[#166235823](https://www.pivotaltracker.com/n/projects/2321088/stories/166235823)

#### Screenshots(if appropriate)
<img width="527" alt="Screenshot 2019-05-24 at 13 49 08" src="https://user-images.githubusercontent.com/37370926/58324003-46a00b80-7e2e-11e9-997f-6de71a4d8d7d.png">
<img width="1440" alt="Screenshot 2019-05-24 at 13 49 17" src="https://user-images.githubusercontent.com/37370926/58324017-53246400-7e2e-11e9-88b4-4d717db24b83.png">
